testing/kfctl/kfctl_go_test.py

```diff
@@ -211,10 +214,26 @@ def test_build_kfctl_go(app_path, project, use_basic_auth, use_istio, config_pat
   ],
                    cwd=app_path)
 
+  # Create $HOME/.kubeconfig file for kfctl to consume.
+  # util.run(["gcloud", "container", "clusters", "get-credentials", "kubeflow-testing", "--zone", zone])
+
+  # We need to use retries because if we don't we see random failures
+  # where kfctl just appears to die.
+  #
   # Do not run with retries since it masks errors
   util.run([kfctl_path, "apply", "-V", "all"], cwd=app_path)
 
-  verify_kubeconfig(app_path)
+  if not cluster_creation_script:
+    verify_kubeconfig(app_path)
+
+
+def filterSpartakus(spec):
```

Comment:
Probably not needed. This is renamed as filter_sparkatus
https://github.com/kubeflow/kubeflow/blob/77e4ee4a050dcd5d60b79bd28bb7923a065b41b4/testing/kfctl/kfctl_go_test.py#L93

Probably  merge issue.

Reply:
Exactly, this was a merge issue, I will rebase to fix.